### ISSUE TITLE
Note does not stay at its line when document is modified

### ISSUE THREAD

**Issue by younrx** [RefactorSecurity/vscode-security-notes#21]
Hello guys!

First I want to thak you for the job you've made. Your extension turned out to be quite useful and it's a real pleasure to have both review notes and code in the same window. Big thanks for your initiatve :rocket:

Some teammates of mine have tried it and found a few issues. One of them is the following: **notes does not stay in place when the document is modified after the note creation**.

Here's how to reproduce it:
- create a note in a file
- add a few lines above the created note (the note correctly follows its line, i.e. the associated line number increases)
- change vscode tab, like if you would work on another file
- go back to the first tab/file. **Note has moved and is now back to the line number of its creation, and not the one it should be considering the modifications made in this file**.

**Comment by ggabarrin** [comment]
Hey @younrx,

Thanks so much for your feedback. Glad to hear you and your team find the tool useful :)

We've been able to reproduce the issue. We'll further explore this issue, but this might also be a vscode issue as well, instead of an issue on the extension.

Thanks!

**Comment by younrx** [comment]
Yeah that's what I fear... I hope there's a way to patch this ! I'll stay tuned 😉

**Comment by ggabarrin** [comment]
Hey @younrx, we've been trying to reproduce this issue but it seems this may have been already fixed by vscode.

We are closing this issue for now. In case you can still reproduce this behavior, please let us know and we'll re-open it and take a more thorough look.

Thanks!
